reactive/containerd.py

```diff
@@ -144,12 +147,28 @@ def install_containerd():
     """
     status_set('maintenance', 'Installing containerd via apt')
     apt_update()
-    apt_install('containerd', fatal=True)
+    apt_install(CONTAINERD_PACKAGE, fatal=True)
+    apt_hold(CONTAINERD_PACKAGE)
 
     set_state('containerd.installed')
     config_changed()
 
 
+@when('containerd.installed')
+@when_not('containerd.version-published')
+def publish_version_to_juju():
+    """
+    Publish the containerd version to Juju.
+
+    :return: None
+    """
+    version_string = _check_containerd()
+    version = version_string.split()[6].split(b'-')[0].decode()
```

Comment:
What if `version_string` is `None`? This will raise an `AttributeError` and cause the hook to fail, right?

Reply:
Yeah, good spot!  Fixed.